app/helpers/Pagination.php

```diff
@@ -83,4 +111,50 @@ public function getOriginalOrderBy(): ?string {
     return $this->originalOrderBy;
   }
 
+  /**
+   * True if filter of given name is present.
+   */
+  public function hasFilter(string $name)
+  {
+    return array_key_exists($name, $this->filters);
+  }
+
+  /**
+   * Return the filter value.
+   * @throws InternalServerErrorException If the filter is not present.
+   */
+  public function getFilter(string $name)
+  {
+    if (!$this->hasFilter($name)) {
+      throw new InternalServerErrorException("Filter $name is not present.");
+    }
+    return $this->filters[$name];
+  }
+
+  public function getRawFilters()
+  {
+    return $this->filters;
+  }
+
+
+  public function getLocale()
+  {
+    return $this->locale;
+  }
+
+  /**
+   * Find appropriate MySQL Collation for given locale.
+   */
+  public function getLocaleCollation()
+  {
+    static $collations = null;
+    if (!$collations) {
+      // First time initialization ...
+      $collations = [
+        'cs' => 'utf8_czech_ci'
```

Comment:
What about `en` collation? Do you assume that it is default and therefore you do not have to set it?

Reply:
Exactly. But I can add it, if you would like.

Reply:
No, need... just to be sure...